### PR TITLE
fix: Fix initialize services in aries farmework

### LINKED ISSUE #383: Fix initialize services in aries farmework

### PR DESCRIPTION
**Pull request by fqutishat**
closes #383

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/framework/aries/framework.go [hunk 1]
```diff
@@ -220,8 +224,11 @@ func createWallet(frameworkOpts *Aries, ctxProvider api.Provider) error {
 	return nil
 }
 
-func createOutboundDispatcher(frameworkOpts *Aries, ctxProvider dispatcher.Provider) error {
-	var err error
+func createOutboundDispatcher(frameworkOpts *Aries) error {
+	ctxProvider, err := frameworkOpts.Context()
```
Comment (troyronda) [inline pkg/framework/aries/framework.go]: why create the context twice instead of passing in from parent?

**Comment by rolsonquadras** [comment]
@fqutishat I think the semantic prefix should be "fix" instead of "bug".

File: pkg/framework/aries/framework.go (hunk 1 above)
Comment (fqutishat) [reply pkg/framework/aries/framework.go]: 1- Wallet need context 
2- OutboundDispatcher need context that has wallet
3- Services need context that has OutboundDispatcher

Comment (fqutishat) [reply pkg/framework/aries/framework.go]: @troyronda we create issue to address this issue

Comment (troyronda) [reply pkg/framework/aries/framework.go]: It seems like we shouldn't be reusing the same Context() method for everything. The first two seem like special bootstrap steps.

(Re-using Context() for bootstrap is confusing - rather there should be specialized private methods that return only the subset type).

Comment (fqutishat) [reply pkg/framework/aries/framework.go]: @troyronda we will fix in  https://github.com/hyperledger/aries-framework-go/issues/212

Comment (fqutishat) [reply pkg/framework/aries/framework.go]: @troyronda I change it

**Pull request merged**